modules/tinymce/src/plugins/link/test/ts/atomic/NormalizeLinkTest.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Optional } from '@ephox/katamari';
+import { TinyHooks } from '@ephox/mcagar';
+import { Hierarchy, SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+import Editor from 'tinymce/core/api/Editor';
+import { getSelectedLink } from 'tinymce/plugins/link/core/NormalizeLink';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.plugins.link.NormalizeLink', () => {
```

Comment:
This entire file should be in the `browser` directory since it's not an atomic test. Atomic tests are things that you can test without a browser not exactly sure why it's called that though but that is the pattern we have.

Reply:
updated